src/faldbt/parse.py

```diff
@@ -46,5 +47,24 @@ def get_dbt_results(project_dir: str, config: RuntimeConfig) -> RunResultsArtifa
         raise FalParseError("Did you forget to run dbt run?") from exc
 
 
-def get_scripts_list(project_dir: str) -> List[Path]:
+def get_scripts_list(project_dir: str) -> List[str]:
     return glob.glob(os.path.join(project_dir, "**.py"), recursive=True)
+
+
+def get_global_script_configs(source_dirs: List[Path]) -> List[str]:
+    global_scripts = []
+    for source_dir in source_dirs:
+        schema_files = glob.glob(os.path.join(source_dir, "**.yml"), recursive=True)
+        global_scripts = []
+        for file in schema_files:
+            schema_yml = load_yaml(file)
+            if schema_yml is not None:
+                fal_config = schema_yml.get("fal", None)
+                if fal_config is not None:
+                    script_paths = fal_config.get("scripts", [])
+                    global_scripts.append(script_paths)
```

Comment:
you could avoid the `_flatten` call by doing `global_scripts += script_paths`

Reply:
lemme try